components/service/firefox-accounts/src/test/java/mozilla/components/service/fxa/FxaAccountManagerTest.kt

```diff
@@ -1886,6 +1887,180 @@ class FxaAccountManagerTest {
         verify(syncManager, times(2)).stop()
         verifyNoMoreInteractions(syncManager)
     }
+    @Test
+    fun `beginAuthenticationAuthSync handles any pairingUrl correctly`() {
+        // Initialize things so the test can run
+        WorkManagerTestInitHelper.initializeTestWorkManager(testContext)
+
+        // Build our dependencies for creating an account manager
+        val tServerConfig = ServerConfig(
```

Comment:
nit: var name prefixes aren't really a convention in Kotlin, maybe just `serverConfig`? or `testServerConfig` if you want to spell it out.

Reply:
I have refactored all those variables to use 'test' instead of 't' as the prefix.